Mathlib/AlgebraicTopology/DoldKan/Decomposition.lean

```diff
@@ -102,9 +102,9 @@ variable {X} {n : ℕ} {Z Z' : C} (f : MorphComponents X n Z) (g : X' ⟶ X) (h
 
 /-- The morphism `X _[n+1] ⟶ Z ` associated to `f : MorphComponents X n Z`. -/
 def φ {Z : C} (f : MorphComponents X n Z) : X _[n + 1] ⟶ Z :=
-  PInfty.f (n + 1) ≫ f.a + ∑ i : Fin (n + 1), (P i).f (n + 1) ≫ X.δ i.revPerm.succ ≫
-    f.b (Fin.revPerm i)
-#align algebraic_topology.dold_kan.morph_components.φ AlgebraicTopology.DoldKan.MorphComponents.φ
+  PInfty.f (n + 1) ≫ f.a + ∑ i : Fin (n + 1), (P i).f (n + 1) ≫ X.δ i.rev.succ ≫
+    f.b (Fin.rev i)
+#align algebraic_topology.dold_kan.morph_components.φ AlgebraicTopology.DoldKan.MorphComponents.φₓ
```

Comment:
```suggestion
#align algebraic_topology.dold_kan.morph_components.φ AlgebraicTopology.DoldKan.MorphComponents.φ
```
These `ₓ` are not needed in the align statements. Because we in fact want mathport translation to pick up on exactly these renames.
Could you please remove them throughout?

Reply:
Mathport will still translate correctly in the presence of ₓ; it just leaves it as a marker in the output translation to indicate a place where the replacement is not an exact match.